x-pack/plugins/rollup/public/crud_app/sections/job_create/steps/step_metrics.js

```diff
@@ -188,10 +188,10 @@ export class StepMetricsUi extends Component {
               <p>
                 <FormattedMessage
                   id="xpack.rollupJobs.create.stepMetricsDescription"
-                  defaultMessage={`
-                    Select the metrics that should be collected while rolling up data. By default,
-                    only the doc_counts are collected for each group.
-                  `}
+                  defaultMessage={
```

Comment:
defaultMessage="Select the metrics that should be collected while rolling up data. By default,
only the doc_counts are collected for each group."